core/templates/pages/skill-editor-page/modal-templates/add-worked-example-modal.controller.ts

```diff
@@ -22,12 +22,18 @@ require(
 
 angular.module('oppia').controller('AddWorkedExampleModalController', [
   '$controller', '$scope', '$uibModalInstance',
-  function($controller, $scope, $uibModalInstance) {
+  'MAX_CHARS_IN_SKILL_WORKED_EXAMPLE_EDITOR',
+  function(
+      $controller, $scope, $uibModalInstance,
+      MAX_CHARS_IN_SKILL_WORKED_EXAMPLE_EDITOR) {
     $controller('ConfirmOrCancelModalController', {
       $scope: $scope,
       $uibModalInstance: $uibModalInstance
     });
 
+    $scope.MAX_CHARS_IN_SKILL_WORKED_EXAMPLE_EDITOR =
```

Comment:
Ditto: avoid attaching this to the template scope if it is not used directly in the template HTML.

Reply:
@kevintab95 Done !